### DISCUSSION TITLE
How do I get a tweet a mention is in reply to?

### DISCUSSION THREAD

**Discussion by AdamVentura** [tweepy/tweepy#1480] [Questions] [answered]
I am trying to create a Twitter bot using Tweepy that replies to mentions but needs to process the tweet the mention is replying to. I have everything else figured out except how to get the tweet the mention is replying to. I have looked through the documentation several times and have been unable to find a way to do this. Any help is much appreciated.

**Comment by Harmon758** [comment, marked as answer]
You can use the `in_reply_to_status_id` or `in_reply_to_status_id_str` attributes/properties of the Status/[Tweet object](https://developer.twitter.com/en/docs/twitter-api/v1/data-dictionary/object-model/tweet) and retrieve it by using [`API.get_status`](https://tweepy.readthedocs.io/en/latest/api.html#API.get_status).

**Reply by Harmon758** [reply]
I'm not sure what you mean. That is how you retrieve the Tweet being replied to; it's unrelated to how you reply to a Tweet.
The `in_reply_to_status_id` and `in_reply_to_status_id_str` attributes/properties will be the ID of the Tweet that's being replied to.

**Reply by tttjj80** [reply]
@AdamVentura Hi,I want to confirm whether you were able to resolve the issue. If so, could you please share how you did it? Thank you very much.
